cli-commands/compiler/compiler.js

```diff
@@ -15,7 +16,13 @@ const performCompilation = (defaultPath) => {
     };
 
     compiler.compile(compileOptions, (error, artifacts, paths) => { 
-        
+        if (!error) {
```

Comment:
The general rule of thumb is to check for erroneous conditions. Id write this if(error) {the error console}, otherwise the success console